lib/classes/eik-config.js

```diff
@@ -61,6 +120,36 @@ module.exports = class EikConfig {
         assert.eikJSON(this[_config]);
     }
 
+    /**
+     * @returns {Promise<FileMapping[]>}
+     */
+    async mappings() {
+        const files = normalizeFilesDefinition(this.files);
+        const resolvedFiles = await resolveFiles(files, this.cwd);
+
+        return resolvedFiles.flatMap(([pathname, file, filePaths]) => {
```

Comment:
Would `destination`, `source` and `sourceFilePaths` be better names here?

Reply:
yeah, I think that they probably would